content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -0,0 +1,259 @@
+set to `NotRequired` for all containers in the pod.
+{{< /note >}}
+
+Below example shows a pod whose container's CPU can be resized without restart, but
+memory resize memory requires the container to be restarted.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: qos-demo
+  namespace: qos-example
+spec:
+  containers:
+  - name: qos-demo-ctr
+    image: nginx
+    resizePolicy:
+    - resourceName: cpu
+      restartPolicy: NotRequired
+    - resourceName: memory
+      restartPolicy: RestartContainer
+    resources:
+      limits:
+        memory: "200Mi"
+        cpu: "700m"
+      requests:
+        memory: "200Mi"
+        cpu: "700m"
+```
+
+{{< note >}}
+In the above example, if desired requests or limits for both CPU _and_ memory
+have changed, the container will be restarted in order to resize its memory.
+{{< /note >}}
+
+<!-- steps -->
+
+
+## Create a pod with resources
+
+You can create a Guaranteed or Burstable [Quality of Service](/docs/tasks/configure-pod-container/quality-service-pod/)
+class pod by specifying requests and/or limits for a pod's containers.
+
+Consider the following manifest for a pod that has one container.
+
+{{< codenew file="pods/qos/qos-pod.yaml" >}}
+
+Create the pod in `qos-example` namespace:
+
+```shell
+kubectl create namespace qos-example
+kubectl create -f https://k8s.io/examples/pods/qos/qos-pod.yaml
+```
+
+This is classified as a Guaranteed QoS class pod requesting 700m CPU and 200Mi
+memory.
+
+View detailed information about the pod:
+
+```shell
+kubectl get pod qos-demo --output=yaml --namespace=qos-example
+```
+
+Also notice that the values of `resizePolicy[*].restartPolicy` defaulted to
+`NotRequired`, indicating that CPU and memory can be resized while container
+is running.
+
+```yaml
+spec:
+  containers:
+    ...
+    resizePolicy:
+    - resourceName: cpu
+      restartPolicy: NotRequired
+    - resourceName: memory
+      restartPolicy: NotRequired
+    resources:
+      limits:
+        cpu: 700m
+        memory: 200Mi
+      requests:
+        cpu: 700m
+        memory: 200Mi
+...
+  containerStatuses:
+...
+    name: qos-demo-ctr
+    ready: true
+...
+    allocatedResources:
+      cpu: 700m
+      memory: 200Mi
+    resources:
+      limits:
+        cpu: 700m
+        memory: 200Mi
+      requests:
+        cpu: 700m
+        memory: 200Mi
+    restartCount: 0
+    started: true
+...
+  qosClass: Guaranteed
+```
+
+
+## Updating the pod's resources
+
+Let's say the CPU requirements have increased, and '800m' CPU is now desired. This
+is typically determined, and may be programmatically applied, by an entity such as
+[VerticalPodAutoscaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler#readme) (VPA).
+
+{{< note >}}
+While you can change a pod's requests and limits to express new desired
+resources, you cannot change the QoS class in which the pod was created.
```

Comment:
```suggestion
resources, you cannot change the QoS class in which the Pod was created.
```

Reply:
This is a weird behavior...

If a container's limits is higher than its requests, the Pod should have been (re)classified as a BestEffort Pod.
But according to the above statement, that won't happen. Please double confirm.